client/app/homePortal/homePortal.js

```diff
@@ -71,11 +46,55 @@ angular.module('dronePass.homePortal', [])
       }
     }
   });
+   /***************** Utilities ***********************/
 
-  $scope.featureCollection = $scope.geojson.data.features;
+  var featureCollection = $scope.geojson.data.features;
```

Comment:
if you are just don't want to type $scope every time but maybe need it in the scope in the future, you can do `var featureCollection = $scope.featureCollection = $scope.geojson.data.features;`